conduct.md

```diff
@@ -10,7 +10,42 @@
 * Overall Conduct
 * Attendance
 * Deadlines
+
+Deadlines should be made by the team leaders of the project and there should be
+multiple deadlines before the final deadline in order to make sure that everything
+is preceding as they should
+
+Acceptable Behavior (A):
+Members shall:
+
+Talk with team members and discuss work needed to be done.
```

Comment:
I would make sure that you are following the same format as other groups when listing what to follow. For example, there should be clear indentation and bullets of when a new point starts under a specific title.